README.md

```diff
@@ -12,6 +12,23 @@ uv sync
 uv pip install -e .
 ```
 
+Build the JS/CSS:
+
+```
+npm run build
+```
+
+If you want to be able to log in (you don't have to; preview covers most things),
```

Comment:
non-blocking

```suggestion
In most cases, you don't need to actually log in locally.
In those cases you can set the `PUDL_VIEWER_LOGIN_DISABLED` environment variable to `true`.
This will let you access the whole site,
including things that are usually gated behind login like the table previews.

But if you're testing something related to the login flow, you will need to set that up:
```